msteams-platform/concepts/build-and-test/deep-link-teams.md

```diff
@@ -164,7 +164,7 @@ Use the following deep link format can be used in a bot, connector, or message e
 
 The query parameters are:
 
-* `fileId`: Unique file ID from Sharepoint Online, also known as `sourcedoc`. For example, `1FA202A5-3762-4F10-B550-C04F81F6ACBD`.
+* `fileId`: Unique file ID from Sharepoint Online, also known as `sourcedoc` or `sharepointIds.listItemUniqueId` from Sharepoint Online or `driveItem` API response. For example, `1FA202A5-3762-4F10-B550-C04F81F6ACBD`.
```

Comment:
**or** `driveItem` API response
Can you please remove the extra "or" above.

Reply:
Updated